protocol/messenger_communities.go

```diff
@@ -365,6 +366,17 @@ func (m *Messenger) handleCommunitiesSubscription(c chan *communities.Subscripti
 						m.logger.Error("failed to save data and prepare response")
 					}
 
+					// if control node changed, the client will be kicked from the community
+					// if owner token will be mint, the client will stay the member of the community
+					if communityResponse.Changes.ShouldMemberLeave {
```

Comment:
This will be a bit more explicit.

```suggestion
					if communityResponse.Changes.ControlNodeChanged != nil && communityResponse.Changes.ShouldMemberLeave {
```